wire-ios/Wire-iOS/Sources/UserInterface/GroupDetails/GroupDetailsViewController.swift

```diff
@@ -116,6 +123,12 @@ final class GroupDetailsViewController: UIViewController, ZMConversationObserver
         })
     }
 
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+
+        setupNavigatiomItem()
```

Comment:
I think we discussed this already but I'm not sure. It won't work if we change if the current interface style is different from the previous one and then call setupNavigatiomItem right?

Reply:
yes